sql_generators/mobile_kpi_support_metrics/templates/engagement.query.sql

```diff
@@ -1,4 +1,24 @@
 {{ header }}
+WITH device_manufacturer_counts AS (
+  SELECT
+    submission_date,
+    device_manufacturer,
+    COUNT(*) AS device_manufacturer_count,
```

Comment:
how about adding a rank and doing top 100?
2k seems arbitrary and I wonder how it'll change over time.

Reply:
Honestly, I did not think about it at the time. But that sounds like a fair suggestion. I can make that change.

Reply:
ok, I made the change. I believe this is what you meant.